fdbclient/NativeAPI.actor.cpp

```diff
@@ -545,11 +545,17 @@ DatabaseContext::DatabaseContext(Reference<Cluster> cluster, Reference<AsyncVar<
 
 DatabaseContext::DatabaseContext( const Error &err ) : deferredError(err), latencies(1000), readLatencies(1000), commitLatencies(1000), GRVLatencies(1000), mutationsPerCommit(1000), bytesPerCommit(1000) {}
 
-ACTOR static Future<Void> monitorClientInfo( Reference<AsyncVar<Optional<ClusterInterface>>> clusterInterface, Reference<ClusterConnectionFile> ccf, Reference<AsyncVar<ClientDBInfo>> outInfo, Reference<AsyncVar<int>> connectedCoordinatorsNumDelayed ) {
-	try {
+ACTOR static Future<Void> monitorClientInfo(Reference<AsyncVar<Optional<ClusterInterface>>> clusterInterface,
+                                            Reference<AsyncVar<Reference<ClusterConnectionFile>>> async_ccf,
```

Comment:
Minor style comment, but we don't generally use underscores in variable names (although like any other style rule, this hasn't been followed consistently).

Reply:
Done